src/mlib/str.h

```diff
@@ -587,6 +591,24 @@ mstr_eq (mstr_view left, mstr_view right)
    return memcmp (left.data, right.data, left.len) == 0;
 }
 
+/**
+ * @brief Determine whether two strings are equivalent ignoring case.
+ */
+static inline bool
+mstr_eq_ignore_case (mstr_view left, mstr_view right)
+{
+#ifdef _WIN32
+#define _mstr_strncasecmp _strnicmp
+#else
+#define _mstr_strncasecmp strncasecmp
+#endif
+
+   if (left.len != right.len) {
+      return false;
+   }
+   return _mstr_strncasecmp (left.data, right.data, left.len) == 0;
```

Comment:
The `undef` of `_mstr_stcncasecmp` is missing.

Reply:
Oops, good eye.